com.github.babluboy.bookworm.json

```diff
@@ -0,0 +1,104 @@
+{
+    "id": "com.github.babluboy.bookworm",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.24",
```

Comment:
Does this also work on 3.26? Any reason why the older runtime?

Reply:
Bookworm is built for elementary OS specifically. Elementary OS runs on Ubuntu LTS, so it is still running 3.24. Once the next release of elementary comes out(in April), this can be changed to a current GNOME runtime.

Reply:
OK - that's a perfectly good reason and indeed uses one of the strengths of flatpak.